kn/fotos/entities.py

```diff
@@ -408,6 +409,12 @@ def update_metadata(self, parent, save=True):
 
         return True
 
+    def set_rotation(self, rotation, save=True):
+        self.rotation = rotation
+        self._data['caches'] = [] # invalidate cache
```

Comment:
Nee, dit is alleen voor de foto's van een andere grootte (`thumb`, `thumb2x`, `large`, `large2x`). Die moeten opnieuw gegenereerd worden na het roteren.